test/timezones/rounding.jl

```diff
@@ -0,0 +1,253 @@
+utc = FixedTimeZone("UTC")
+fixed = FixedTimeZone("UTC-06:00")
+winnipeg = TimeZone("America/Winnipeg")             # UTC-6:00 (UTC-5:00 during DST)
+st_johns = TimeZone("America/St_Johns")             # UTC-3:30 (UTC-2:30 during DST)
+eucla = TimeZone("Australia/Eucla")                 # UTC+8:45
+colombo = TimeZone("Asia/Colombo")                  # See note below
```

Comment:
Oh yes, you prefer calls to `resolve`. It's been a while since I wrote anything for `TimeZones`. I think I can figure it out.